SportsHub/SportsHub.DAL/Data/Configurations/ArticleEntityTypeConfiguration.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using SportsHub.DAL.Data.Configurations.Constants;
+using SportsHub.Domain.Models;
+
+namespace SportsHub.DAL.Data.Configurations
+{
+    public class ArticleEntityTypeConfiguration : IEntityTypeConfiguration<Article>
+    {
+        public void Configure(EntityTypeBuilder<Article> article)
+        {
+            article
```

Comment:
The title should have a constraint on the length. For all string types better to have limitations for the database!